tests/dapr_tests.mk

```diff
@@ -209,6 +209,12 @@ test-e2e-all: check-e2e-env test-deps
 	# Note2: use env variable DAPR_E2E_TEST to pick one e2e test to run.
```

Comment:
We can use this one.

Reply:
Yes, but the env param `DAPR_E2E_TEST ` only picks all e2e apps or one e2e app, it can't pick multiple e2e apps at once.

Reply:
A target for one or all is common, the rest often write in their own script.